lib/ui/src/main/scala/net/wiringbits/ui/components/core/widgets/ValidatedTextInput.scala

```diff
@@ -14,7 +14,7 @@ abstract class ValidatedTextInput[T: TextValidator] {
       field: FormField[T],
       disabled: Boolean = false,
       onChange: ValidationResult[T] => Unit,
-      margin: PropTypes.Margin = muiStrings.dense
+      margin: "dense" = "dense"
```

Comment:
We need the potential margin types, is `dense` the only one? if so, we can just remove the argument